Tools/gulp/gulp-nuget-proj-sync.js

```diff
@@ -0,0 +1,65 @@
+// through2 is a thin wrapper around node transform streams
+var through = require('through2');
+var gutil = require('gulp-util');
+var xml2js = require('xml2js');
+var path = require('path');
+var glob = require('glob');
+var fs = require('fs');
+var PluginError = gutil.PluginError;
+
+const PLUGIN_NAME = 'gulp-nuget-proj-sync';
+
+function gulpNugetProjSync(opts) {
+  // Creating a stream through which each file will pass
+  return through.obj(function(file, enc, cb) {
+    var baseDir = path.dirname(file.path);
+    var nuprojPathes = glob.sync(path.join(baseDir, '../*.nuget.proj'));
+
+    var nuprojPath = null;
+    if (!nuprojPathes || nuprojPathes < 1) {
```

Comment:
sorry for my lack of node.js knowledge, but i don't think "nuprojPathes < 1" helps to deal with any new condition, unless the 'nuprojPathes' can be a float number between 0 and 1